aws/logs_monitoring/template.yaml

```diff
@@ -539,11 +559,11 @@ Resources:
       Description: Datadog API Key
       SecretString:
         Ref: DdApiKey
-  ForwarderZipsBucket:
+  ForwarderBucket:
     Type: AWS::S3::Bucket
     Properties:
       BucketEncryption:
-          ServerSideEncryptionConfiguration:
+        ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
```

Comment:
See https://github.com/DataDog/datadog-serverless-functions/pull/349, let's change to `SSEAlgorithm: 'aws:kms'`.